src/material/expansion/_expansion-variables.scss

```diff
@@ -1,11 +1,11 @@
 // Default minimum and maximum height for collapsed panel headers.
-$header-collapsed-height: 48px !default;
+$header-collapsed-height: fit-content !default;
```

Comment:
`fit-content` doesn't appear to be supported on Firefox: https://developer.mozilla.org/en-US/docs/Web/CSS/height#browser_compatibility